remoting/src/main/java/org/eclipse/tractusx/agents/remoting/Application.java

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) 2022,2023 Contributors to the Eclipse Foundation
+//
+// See the NOTICE file(s) distributed with this work for additional
+// information regarding copyright ownership.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Apache License, Version 2.0 which is available at
+// https://www.apache.org/licenses/LICENSE-2.0.
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations
+// under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+package org.eclipse.tractusx.agents.remoting;
+
+import org.eclipse.rdf4j.common.iteration.Iterations;
+import org.eclipse.rdf4j.query.BindingSet;
+import org.eclipse.rdf4j.query.QueryLanguage;
+import org.eclipse.rdf4j.query.TupleQuery;
+import org.eclipse.rdf4j.query.TupleQueryResult;
+import org.eclipse.rdf4j.repository.Repository;
+import org.eclipse.rdf4j.repository.RepositoryConnection;
+import org.eclipse.rdf4j.repository.sail.SailRepository;
+import org.eclipse.tractusx.agents.remoting.config.*;
+
+import java.util.Arrays;
+
+
+/**
+ * A sample main application which exposes an RDF4J-SparQL endpoint
+ * with a fixed binding. Just to demonstrate how to build custom servers.
+ * Usually, this library is added to the rdf4j lib path and the
+ * endpoints/repositories using the function binding are then defined
+ * declaratively.
+ */
+public class Application {
+
+
+    @Override
+    public String toString() {
+        return super.toString()+"/application";
+    }
+
+    /**
+     * main logic sets up the rdf4j server programatically.
+     * @param args command line args
+     */
+
+    public static void main(String[] args) {        
+        RemotingSailConfig rsc=new RemotingSailConfig(RemotingSailFactory.SAIL_TYPE);
+        ServiceConfig ic=new ServiceConfig();
+        rsc.putService("https://www.w3id.org/catenax/ontology/prognosis#Invocation",ic);
+        ic.setTargetUri("class:io.catenax.knowledge.agents.remoting.TestFunction#test");
+        ArgumentConfig ac=new ArgumentConfig();
+        ac.setArgumentName("arg0");
+        ic.getArguments().put("https://www.w3id.org/catenax/ontology/prognosis#input-1",ac);
```

Comment:
This URL redirect service is pointing to catenax-ng GH org. Do not link/reference catenax-ng repositories and/or GH org.

This comment addresses all appearances of this redirect service.

Reply:
1. Catenax will stay as an assocation
2. It will have control over particular standards, schemas and ontologies (which tractusx may use in order to provide implementations according to these standards, schemas and ontologies).
3. Using a redirection service to make the actual location of these standards, schemas and ontologies is best practice.